dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -246,10 +246,12 @@ def present_level
       script_level_id: @script_level.id,
       level_id: @level.id)
 
-    @sublevel_callback = milestone_script_level_url(
-      user_id: current_user.try(:id) || 0,
-      script_level_id: @script_level.id,
-      level_id: '') if @level.game.level_group?
+    if @level.game.level_group? || @level.try(:contained_levels).present?
+      @sublevel_callback = milestone_script_level_url(
+        user_id: current_user.try(:id) || 0,
+        script_level_id: @script_level.id,
+        level_id: '')
```

Comment:
The `level_id` - it is being provided as a partial callback URL here (with the `level_id` blank) so that the app code can append the sublevel id to this `sublevel_callback` as RESTful parameter.